src/main/java/dude/exception/DudeTaskNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package dude.exception;
+
+/**
+ * Thrown to indicate that no task with matching prompt is found in TaskList.
+ */
+public class DudeTaskNotFoundException extends DudeException {
+
```

Comment:
Should there be an empty line here? Spotted this in other areas as well.

Reply:
This isn't needed, but out of habit, I tend to leave a blank line before any method. It doesn't violate any rules when running `checkstyleMain`. I hadn't noticed this before., not sure whether this should be avoided or not.